### PR TITLE
DIT-2880 made the max date inclusive

### PR DESCRIPTION
**Pull request by oliwiaKot**

### CODE REVIEW

File: app/models/forms/ReportDateForm.scala [hunk 1]
```diff
@@ -41,6 +41,13 @@ object ReportDateForm {
       .toInstant
   }
 
+  private val formDate2InstantAtEndOfDay: DateForm => Instant = { dateForm =>
+    LocalDate
+      .of(dateForm.year.toInt, dateForm.month.toInt, dateForm.day.toInt + 1)
+      .atStartOfDay(ZoneOffset.UTC)
+      .toInstant
```
Comment (DavidGregory084) [inline app/models/forms/ReportDateForm.scala:48]: I think we need to be careful here - `dateForm.day.toInt + 1` could produce an invalid date even if the user enters something valid, e.g. if they entered the 31st of a month. How about using `plusDays` instead:

```suggestion
    LocalDate
      .of(dateForm.year.toInt, dateForm.month.toInt, dateForm.day.toInt)
      .plusDays(1)
      .atStartOfDay(ZoneOffset.UTC)
      .toInstant
```

**Review by DavidGregory084 — CHANGES_REQUESTED** [review]
Looks good, just a comment about how we are incrementing the date 👍

File: app/models/forms/ReportDateForm.scala (hunk 1 above)
Comment (oliwiaKot) [reply app/models/forms/ReportDateForm.scala]: thank you! I've applied that now

**Review by DavidGregory084 — APPROVED** [review]
👍

**Review by vajarova — APPROVED** [review]

**Pull request merged**